src/dotnet/APIView/APIViewWeb/Client/src/shared/comments.ts

```diff
@@ -110,6 +111,8 @@ $(() => {
   });
 
   $(document).on("submit", "form[data-post-update='comments']", e => {
+    disableButtonTemp('button.comment-submit-button', 3000);
```

Comment:
This does not seem to work as it disables the button for the next reply to the comment. Rather than using the class selector to select the comment (which will apply to all instances of that button) you can trigger the function for only the button that triggered the submit event by aiming you selection off the event (e) object (Should be possible since the button is in the form). And if you can do that you don't have to worry about re-enabling it since it will apply to only that instance of the comment form / button.